cmd/troubleshoot.go

```diff
@@ -112,7 +114,7 @@ func (status *Status) check() (string, error) {
 	status.Version = newVersionStatus(status.cli)
 	status.System = newSystemStatus()
 	status.Configuration = newConfigurationStatus(status)
-	status.APIReachability = newAPIReachabilityStatus(status.cfg.GetString("apibaseurl"))
+	status.APIReachability = newAPIReachabilityStatus(status.cfg)
```

Comment:
Passing the config here seems a bit heavy handed. What if instead of the original value you passed `config.InferSiteURL(v.GetString("apibaseurl"))`? Would that not achieve the same thing?

Reply:
No, the site URL is not the same as the API, so it would call https://exercism.io/ping (which would fail with a 404).